source/nodejs/adaptivecards-designer/src/open-image-dialog.ts

```diff
@@ -0,0 +1,324 @@
+import { Dialog } from "./dialog";
+
+export class FetchManager {
+
+    async getSampleImages() {
+        const option ={
+            method: "GET",
+            headers: {
+                "Content-Type": "application/json",
+                "Access-Control-Allow-Origin": "*"
+            }
+        }
+        let sampleImages = await fetch(process.env.PIC_TO_CARD_PREDICTION_API+"/get_card_templates", option);
+        return await sampleImages.json();
+    }
+
+    async getPredictedData(imageContent, isTemplateBindingEnabled) {
+        const options = {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
```

Comment:
common headers can be moved outside as a constant.

Reply:
Addressed the review comments